.gitignore

```diff
@@ -5,7 +5,7 @@
 *.dll
 *.so
 *.dylib
-bin
+/bin
```

Comment:
seems bin/* would be also be ok for the case you mentioned, i hit it previously with ovn-operator and kept it similar pattern as testbin/*.

Reply:
Yeah. That also works.

The reason I used `/bin` was it is used by a few other operators such as cinder/glance/nova and tried to make this consistent across operators.

Reply:
Ok Thanks, that works for me.